library/src/commonMain/kotlin/com/gu/recipe/ScaleRecipe.kt

```diff
@@ -52,15 +54,14 @@ internal fun scaleTemplate(template: ParsedTemplate, factor: Float): String {
         when (element) {
             is TemplateElement.TemplateConst -> element.value
             is TemplateElement.QuantityPlaceholder -> {
+                val max = if (element.min != element.max) element.max else null
                 val (scaledMin, scaledMax) = if (element.scale) {
                     val scaledMin = (element.min * factor)
-                    val scaledMax = if (element.min != element.max) {
-                        element.max?.let { (it * factor) }
-                    } else null
+                    val scaledMax = max?.let { (it * factor) }
```

Comment:
When `max` has no value then `scaleMax` would be null - would be safer to set `1` as fallback to `scaledMax`?